packages/blocks/ternary/src/lib/actions/get-budgets.ts

```diff
@@ -20,10 +21,9 @@ export const getBudgets = createAction({
         },
       });
       return response.body as any[];
-    } catch (e) {
-      console.error('Error getting budgets!');
-      console.error(e);
-      return e;
+    } catch (error) {
+      logger.error('Error getting budgets.', error);
```

Comment:
I'm not sure we need to log it, there is nothing we can do with this when we receive it in logzio